.gitignore

```diff
@@ -0,0 +1,15 @@
+# Compiled Bytes
+__pycache__/
```

Comment:
Remove this file from the commit.

Reply:
Without the gitignore, it is adding the bazel symlinks on git add -A
Else, I need to add the files one by one

Reply:
You can keep the file locally, just remove it from the commit.

Reply:
Removed :)